backend/download/tests/test_download_views.py

```diff
@@ -231,6 +232,7 @@ def test_download_with_tag(db, admin_client, small_mock_corpus, index_small_mock
 
 
 def test_unauthenticated_download(db, client, basic_mock_corpus, basic_corpus_public, index_basic_mock_corpus):
+    cache.clear()  # clear cache to avoid throttle limit
```

Comment:
Since this is repeated several times, perhaps you could create a `clear_cache` fixture for these tests?

I think it would be fine to add an autouse to the fixture as well: there should never be side effects from the cache being preserved between unit tests.

Reply:
Yeah good point. I'll use an auto-use fixture so we don't need to repeat this code for any future tests.